src/utils.js

```diff
@@ -1,22 +1,44 @@
 const CALLBACK_NAME = '_grecaptchaonloadcallback'
 
+let loadScriptPromise
```

Comment:
You could initiliaze it with null.

```js
let loadScriptPromise = null
```

Reply:
I initialized it with `undefined` to be explicit, but the linter stripped that out. I just tried using `null` instead and the linter left that in (I'm not surprised) so I'll change to that.